doc/source/contributing/contribution_guidelines.rst

```diff
@@ -73,7 +75,8 @@ General Guidelines
 
 Basic Steps to Contribute
 ^^^^^^^^^^^^^^^^^^^^^^^^^
-We encourage users to follow the `git pull request workflow <https://www.asmeurer.com/git-workflow/>`_. For more detailed steps, please see :ref:`How to Contribute<dev_guide_label>`.
+We encourage users to follow the `git pull request workflow <https://www.asmeurer.com/git-workflow/>`_.
+For more detailed steps, please see :ref:`How to Contribute<dev_guide_label>`.
```

Comment:
For ReadtheDocs pages, is it best practice to place sentences on separate lines like this?

Reply:
As with most things, putting each sentence on a new line for rendered text (One Sentence Per Line, or OSPL) as "best practice" is up for debate. It requires that whatever system you are ultimately using to render your text treats a single newline as a sentence break and has a separate new paragraph indicator (for markdown and rst, it's simply two consecutive newlines). Personally, I've found that for text in shared version controlled workflows (e.g. Jupyterbooks, documentation), it reduces the cognitive load of reviewing text changes if OSPL is followed. For instance, if one contributor automatically reformats text to follow PEP's 79 character limit and another reformats using the (also widely accepted) 99 character limit, you end up with a diff that - instead of highlighting actual text changes - is effectively only highlighting where the hard returns are (vs where they used to be). Or you end up with edited text that has really uneven line lengths because it's not being reformatted. Having spent time reviewing text in these situations where it becomes really unclear what is an actual text edit, I think starting from OSPL is a good practice to help minimize this, since at least some proportion of the sentences will be shorter than these character widths. icepyx hasn't adopted any approach explicitly, but I try to use natural sentence breaks (e.g. commas) and new sentences on newlines as a starting point. This was (obviously) not my original approach, so as I make edits to text, I'm trying to adopt the OSPL practice.